index.html

```diff
@@ -41,7 +50,196 @@ <h1 class="hero_head">Hey there. I'm Jose Delphin.<br> <span>I'm a software deve
       <a id="down_icon" href="#"><img src="assets/images/down.png" alt="Icon for moving"></a>
     </section>
   </div>
- 
+  <header>
+    <h2 id="recent-works" class="works_head">My Recent Works </h2>
+    <img class="center" src="assets/images/Indicator.png" alt="Design line">
+
+  </header>
+
+  <!-- main grid container -->
+  <section id="project_section" class="grid_container">
+    <ul class="flex_container_secondary">
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
+          <small class="collapse"></small>
+        </div>
+      </li>
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
```

Comment:
I suggest using just one `h3` tag here and a `br` to achieve text wrapping to the next line because currently, the spacing is too much on the client side.

Your design 👇👇👇


<img width="300" alt="margin prob" src="https://user-images.githubusercontent.com/84930355/205422873-f5285bcd-d4a4-477a-9c4a-ed5885323949.png">





Figma template 👇👇👇



<img width="263" alt="fig marg prob" src="https://user-images.githubusercontent.com/84930355/205422886-90f14039-b56b-4873-aa7c-0b90a597aa61.png">

Reply:
Great, you are right. It is done

Reply:
Thank you